lib/rules/no-multi-spaces.js

```diff
@@ -83,6 +83,34 @@ module.exports = {
             return false;
         }
 
+        /**
+         * Formats value of given comment token for error message by truncating its length.
+         * @param {ASTNode} token comment token
+         * @returns {string} formatted value
+         * @private
+         */
+        function formatReportedCommentValue(token) {
+            const isBlockComment = token.type === "Block";
+            const valueLines = token.value.split("\n");
+            const value = valueLines[0];
+            const formattedValue = `${value.substring(0, 12)} ...${isBlockComment ? " " : ""}`;
+
+            return valueLines.length === 1 && value.length <= 12 ? value : formattedValue;
+        }
+
+        /**
+         * Creates a fix function that removes the multiple spaces between the two tokens
+         * @param {RuleFixer} leftToken left token
+         * @param {RuleFixer} rightToken right token
```

Comment:
Are these annotations supposed to be `{Token}` instead of `{RuleFixer}`? (I see this was in the code before this change, but it seems like it was a mistake.)